packages/platform-browser/test/browser/bootstrap_spec.ts

```diff
@@ -628,7 +628,11 @@ function bootstrap(
       })();
     });
 
-    it('should remove styles when transitioning from a server render', done => {
+    // TODO: Re-enable this when we replace Domino for our tests.
+    // This test no longer works as Domino does not support complex `querySelectorAll`.
+    // Like `doc.querySelectorAll(`style[ng-transition="${this.appId}"]`)` which is used in the
+    // `DomSharedStylesHost`.
+    xit('should remove styles when transitioning from a server render', done => {
```

Comment:
Can we replace it with the `isBrowser` check?
```suggestion
    isBrowser && it('should remove styles when transitioning from a server render', done => {
```

Reply:
Tried this, but the test ran regardless.

Reply:
Actually, reading through the test. All this test is doing is testing https://github.com/angular/angular/pull/48253/files#diff-ee69a8cb96597d248fd8f6b98b8473f5c56c1d723050c67f6da49832c387fcb2L18 which was removed.